poetry/installation/chef.py

```diff
@@ -81,7 +81,7 @@ def get_cached_archives_for_link(self, link):  # type: (Link) -> List[Link]
         links = []
         for archive_type in archive_types:
             for archive in cache_dir.glob("*.{}".format(archive_type)):
-                links.append(Link("file://{}".format(str(archive))))
+                links.append(Link("file://{}".format(str(archive.as_posix()))))
```

Comment:
```suggestion
                links.append(Link(archive.as_uri()))
```

Reply:
Oh, I was googling around for urlencoding this somehow like this, but I failed to find that this was possible with pathlib. I'm starting to like pathlib more and more 😄 👍